src/scanpy/datasets/_ebi_expression_atlas.py

```diff
@@ -58,31 +61,35 @@ def download_experiment(accession: str):
     )
 
 
-def read_mtx_from_stream(stream: BinaryIO) -> sparse.csr_matrix:
+def read_mtx_from_stream(stream: ReadCsvBuffer[bytes]) -> sparse.csr_matrix:
     curline = stream.readline()
     while curline.startswith(b"%"):
         curline = stream.readline()
-    n, m, _ = (int(x) for x in curline[:-1].split(b" "))
+    n, m, e = map(int, curline[:-1].split(b" "))
 
+    dtype_data = np.float32
```

Comment:
I'm a little lost - so before the referenced PR, the default was float32 and now this just goes back to that?

Reply:
yup!